app/src/main/java/com/ncnf/event/EventRelevanceCalculator.java

```diff
@@ -0,0 +1,84 @@
+package com.ncnf.event;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+
+public class EventRelevanceCalculator {
+
+    private List<PublicEvent> events = new ArrayList<>();
+    private List<Event> sortedList;
```

Comment:
The List could be final and simply updated

Reply:
Done